docs/release-notes/8.16.asciidoc

```diff
@@ -0,0 +1,117 @@
+** This can cause event processing queues to overflow, and lead to dropped events.
+** Many file events won't contain hashes. Hash collection is best effort and not guaranteed to be present in every event. Hashes are collected asynchronously, and shortly after the file activity. Hashes might be missing if the file was rapidly renamed, moved, deleted, or (on Windows) opened by another process without https://learn.microsoft.com/en-us/windows/win32/fileio/creating-and-opening-files[read sharing].
+* Improves {elastic-defend} enabling the use of dynamic {filebeat-ref}/kafka-output.html#topic-option-kafka[topics] for the kafka output.
+* Improves {elastic-defend} by integrating a new Event Tracing for Windows (ETW) provider (Microsoft-Windows-WMI-Activity) to create new event types that can be used by prebuilt endpoint rules to detect malicious WMI activity.
+
+[discrete]
+[[enhancements-8.16.0]]
+==== Enhancements
+* Enables you to open the Rule details flyout from the Alerts table ({kibana-pull}191764[#191764]).
+* Allows you to resize the alert and event details flyouts and choose how it's displayed (over the Alerts table or next to it) ({kibana-pull}192906[#192906], {kibana-pull}182615[#182615]).
+* Allows you to enable entity risk scoring in multiple {kib} spaces ({kibana-pull}192671[#192671]).
+* Creates a new API endpoint for cleaning up entity risk scoring data: `DELETE /api/risk_score/engine/dangerously_delete_data` ({kibana-pull}191843[#191843], {kibana-pull}189872[#189872]). 
+* Removes Elastic AI Assistant's default system prompts. The instructions previously contained in those prompts are now automatically included without user interaction, so Elastic AI Assistant will remain focused on relevant topics. Custom system prompts are still available ({kibana-pull}191847[#191847]). 
+* Improves Elastic AI Assistant's ability to generate {esql} queries ({kibana-pull}195480[#195480], {kibana-pull}188492[#188492]). 
+* Adds a button that lets you quickly add queries generated by Elastic AI Assistant to a rule's definition ({kibana-pull}190963[#190963]).
+* Adds an "Other" option to the OpenAI connector's "Select an OpenAI provider" dropdown menu. Select this option when <<connect-to-byo-llm, connecting to your own custom LLM>> ({kibana-pull}194831[#194831]).
+* Improves Attack Discovery in the following ways ({kibana-pull}195669[#195669]): 
+** Attack Discovery can now process up to 500 alerts (previous maximum: 100). This setting can now be adjusted directly from the Attack Discovery page, and is stored locally instead of in {es}
+** Attack Discovery now combines related discoveries that would previously have appeared separately
+** Attack Discovery now detects and displays an error instead of hallucinated output
+* Adds an **Install and enable** button to the prebuilt rule UI, so you don't have to wait for rules to install before enabling them ({kibana-pull}191529[#191529]). 
```

Comment:
Nit: `prebuilt rule UI` sounds a bit too vague IMHO - we have a few different UI components and pages related to prebuilt rules.

```suggestion
* Adds an **Install and enable** button to the **Add Elastic Rules** page, so you don't have to wait for rules to install before enabling them ({kibana-pull}191529[#191529]). 
```